packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Async, css } from 'office-ui-fabric-react/lib/Utilities';
```

Comment:
Do we need to update the `externals` mapping in tools/pack.js for this?

Reply:
let me check, possibly--seems redundant though

Reply:
you're right, I'll look into adding error-ing out when we are missing OF mappings

Reply:
this is better: https://github.com/Microsoft/roosterjs-react/pull/7